src/js/components/MaskedInput/doc.js

```diff
@@ -84,6 +84,11 @@ export const themeDoc = {
     type: 'string',
     defaultValue: '18px',
   },
+  'maskedInput.disabled.opacity': {
+    description: 'The opacity when the MaskedInput is disabled.',
+    type: 'number',
+    defaultValue: 0.3,
+  },
```

Comment:
I think you are missing the property that will make the connection to the default 0.3 value
```
...themeDocUtils.disabledStyle,
```

Reply:
You're right, it's falling back on global.control.disabled.opacity -- I will update the documentation so the link is clear.